pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -46,14 +46,29 @@ type AccountWatcher struct {
 func initialize(client client.Client, watchInterval time.Duration) *AccountWatcher {
 	log.Info("Initializing the totalAccountWatcher")
 
+	cm := &corev1.ConfigMap{}
+	err := client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if err != nil {
+		log.Error(err, "Failed to get configmap")
+	}
```

Comment:
isn't this what the `GetOperatorConfigMap` function does? What's the reason we can't use it here?